RadixWallet/Clients/AppPreferencesClient/AppPreferencesClient+Interface.swift

```diff
@@ -12,24 +14,20 @@ public struct AppPreferencesClient: Sendable {
 	public var extractProfileSnapshot: ExtractProfileSnapshot
 	public var deleteProfileAndFactorSources: DeleteProfileSnapshot
 
-	public var getDetailsOfSecurityStructure: GetDetailsOfSecurityStructure
```

Comment:
End user has never seen/used SecurityStructure: part of MFA preparation work which will be redone once we have this in Sargon.